src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Argument.cs

```diff
@@ -283,6 +284,32 @@ public override void Action(string argumentValue)
     public override string ToString() => Value ? "true" : "false";
 }
 
+public abstract class EnumPageLoadStrategyArgument : Argument<PageLoadStrategy>
+{
+    private readonly PageLoadStrategy _defaultValue;
+
+    public EnumPageLoadStrategyArgument(string prototype, string description, PageLoadStrategy defaultValue)
+        : base(prototype, description, defaultValue)
+    {
+        _defaultValue = defaultValue;
+    }
+
+    public override void Action(string argumentValue)
+    {
+        if (string.IsNullOrEmpty(argumentValue))
+        {
+            Value = _defaultValue;
+        }
+        else
+        {
+            Value = argumentValue.ToLower().Equals("none") ? PageLoadStrategy.None :
+                argumentValue.ToLower().Equals("eager") ? PageLoadStrategy.Eager :
+                argumentValue.ToLower().Equals("normal") ? PageLoadStrategy.Normal :
```

Comment:
better to use the compare options instead of tolower+equals:

```suggestion
            Value = argumentValue.Equals("none", StringComparison.OrdinalIgnoreCase) ? PageLoadStrategy.None :
                argumentValue.Equals("eager", StringComparison.OrdinalIgnoreCase) ? PageLoadStrategy.Eager :
                argumentValue.Equals("normal", StringComparison.OrdinalIgnoreCase) ? PageLoadStrategy.Normal :
```